src/builder/builder.go

```diff
@@ -61,6 +61,47 @@ func NewFromConfig(conf string) *Builder {
 	return b
 }
 
+// Create a default builder.conf using the active directory as base path for the
+// variables values.
+func (b *Builder) CreateDefaultConfig(builderconf string) {
+	pwd, err := os.Getwd()
+	if err != nil {
+		helpers.PrintError(err)
+		os.Exit(1)
+	}
+
+	if builderconf == "" {
+		builderconf = pwd + "/builder.conf"
+	}
+
+	err = helpers.CopyFile(builderconf, "/usr/share/defaults/bundle-chroot-builder/builder.conf", false)
+	if os.IsExist(err) {
+		fmt.Println("File 'builder.conf' already exists. Skipping...")
+		return
+	} else if err != nil {
+		helpers.PrintError(err)
+		os.Exit(1)
+	}
+
+	fmt.Println("Creating new builder.conf configuration file...")
+
+	raw, err := ioutil.ReadFile(builderconf)
+	if err != nil {
+		helpers.PrintError(err)
+		os.Exit(1)
+	}
+
+	data := strings.Replace(string(raw), "/home/clr/mix", pwd, -1)
+	data += "\n[Mixer]\n"
+	data += "RPMDIR=" + pwd + "/rpms\n"
+	data += "REPODIR=" + pwd + "/local\n"
+
+	re := regexp.MustCompile("<(.)+>\n")
+	data = re.ReplaceAllString(data, "file://"+pwd+"/update/www\n")
```

Comment:
Does this actually work as a valid value for `CONTENTURL` or `VERSIONURL`? 

If I understand correctly, this is the value that gets baked into images, telling the resultant swupd-client where to look for updates. 1) Does swupd-client actually understand the `file://` protocol? And 2) this URL is a path on the build host; even if swupd-client understands `file://`, this wouldn't be a valid path, right?

Reply:
Those values came from here: https://github.com/clearlinux/swupd-client/blob/master/swupd-add-pkg
I figured if this is the template being generated by swupd we should use the same.

Reply:
Ugh sorry, that's a very specific nuanced feature that was just recently added to swupd..I would ignore it & anything related completely for mixer. Swupd does understand file://, but @kevincwells is right that those URL variables get injected into the image so the client points to the server the mix content is hosted on -> which is in all cases, except that weird swupd one, some other remote location.

Reply:
Fair enough. In that case, what should we have as defaults?

Reply:
@tmarcu @kevincwells

Reply:
I don't think this is something you can provide a good default value for. It needs to be the URL at which deployed swupd-clients can reach valid update content -- something equivalent to https://cdn.download.clearlinux.org/update/, but for your derivative mix. The tool cannot predict what that likely is, and hence why it's "&lt;CONTENT URL GOES HERE&gt;" kind of values right now -- something that clearly indicates you need to set it.

If you want the tool to be able to fill these, the best you can likely do is allow the user to pass in the value to `init-mix`.

@tmarcu any better idea?

Reply:
@rchiossi @kevincwells I can't say what a good default is 'cause it depends fully on where the mix content ends up going. The trivial solution is "<current ip address>/update", assuming you host your content on the build machine under the /update folder, wherever your httpd directory is. However it could be anywhere the httpd conf points, the content might get built on one server but rsync'd over to another release server, etc, so it's hard to make a good default for these options.